packages/matter.js/src/behavior/state/transaction/Tx.ts

```diff
@@ -506,6 +506,10 @@ class Tx implements Transaction {
                 () => participant.postCommit?.(),
                 () => postCommitNextParticipant(),
                 error => {
+                    if (error instanceof MatterError) {
```

Comment:
See my comments in Reactors.ts, the same applies here.

Reply:
Reverted